internal/component/prometheus/relabel/relabel.go

```diff
@@ -99,14 +104,22 @@ func New(o component.Options, args Arguments) (*Component, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	debuggingStreamHandler, err := o.GetServiceData(livedebugging.ServiceName)
+	if err != nil {
+		return nil, err
+	}
+	debuggingStreamHandler.(livedebugging.DebugStreamManager).Register(name)
```

Comment:
Instead of creating `const name` can you access name from `opts`?

Reply:
opts does not have just the name, it has the ID which is component name + label